src/Lexer.js

```diff
@@ -80,6 +80,8 @@ var tokenRegex = new RegExp(
     "([ \r\n\t]+)|" +                                 // whitespace
     "([!-\\[\\]-\u2027\u202A-\uD7FF\uF900-\uFFFF]" +  // single codepoint
     "|[\uD800-\uDBFF][\uDC00-\uDFFF]" +               // surrogate pair
+    "|\\\\verb\\*([^\0]).*?(?:\\3|\n|$)" +            // \verb*
```

Comment:
I confirmed that `\0` didn't work in LaTeX, with error message `Text line contains an invalid character.`  But that's probably just TeX being unhappy with the character in general.  I think it's better to allow `\0`, but I didn't know how.  Thanks for the tip!